servlet-bom/build.gradle

```diff
@@ -2,8 +2,7 @@ plugins {
     id "io.micronaut.build.internal.bom"
 }
 
-dependencies {
-    constraints {
-        api("javax.servlet:javax.servlet-api:$servletApiVersion")
-    }
-}
\ No newline at end of file
+micronautBom {
+    // exclude example projects
+    extraExcludedProjects.add("test-kotlin-jetty")
```

Comment:
I think it would be better to rename the project to `test-suite-kotlin-jetty`. If there were any documentation references in asciidoc we would need to update them as well. But by renaming the module, we would not need to exclude it here.

Reply:
👍